### DISCUSSION TITLE
Requests

### DISCUSSION THREAD

**Discussion by lulavc** [anirudhmalik/xhunter#42] [General]
Files up to 100mb able to be used.
Option to bulk download or select folders to download.
Option to use compressed files i.e. .zip and .rar instead of apk.

Im going to hardcore use itt and if u want we can make a group of beta testers.

**Comment by anirudhmalik** [comment]
> Files up to 100mb able to be used. Option to bulk download or select folders to download. Option to use compressed files i.e. .zip and .rar instead of apk.
> 
> Im going to hardcore use itt and if u want we can make a group of beta testers.

I didn't understand what you are requesting. Please elaborate

**Comment by anirudhmalik** [comment]
Closing in favor of v2. See [v2.0-demo](https://github.com/anirudhmalik/xhunter/releases/tag/v2.0-demo) and the repo README. New bugs: open a fresh issue with repro steps. Thanks.
